http/req/doc.go

```diff
@@ -0,0 +1,16 @@
+/*
+Package req provides ergonomics for handling an HTTP request.
```

Comment:
is `req` really doing any other "handling" than parsing? maybe `http/parse` is more explicit?

Reply:
It's validating the data parsed as well, but nothing beyond that yet. Data from an HTTP request being the primary entity managed by this package, `req` felt broad enough to include those 2 functions, but I'm not married to it.